bunk.md

```diff
@@ -0,0 +1,7 @@
+Omar Little: Shoot, the way y'all looking at things, ain't no victim to even speak on.
+Det. William 'Bunk' Moreland: Bullshit, boy. No victim? I just came from Tosha's people, remember? All this death, you don't think it ripples out? You don't even know what the fuck I'm talking about. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.